examples/morpheusvm/tests/workload/workload.go

```diff
@@ -22,78 +21,27 @@ import (
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/consts"
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/vm"
-	"github.com/ava-labs/hypersdk/fees"
-	"github.com/ava-labs/hypersdk/genesis"
+	"github.com/ava-labs/hypersdk/tests/fixture"
 	"github.com/ava-labs/hypersdk/tests/workload"
 )
 
 const (
-	initialBalance  uint64 = 10_000_000_000_000
-	txCheckInterval        = 100 * time.Millisecond
+	TxCheckInterval = 100 * time.Millisecond
 )
 
-var (
-	_              workload.TxWorkloadFactory  = (*workloadFactory)(nil)
-	_              workload.TxWorkloadIterator = (*simpleTxWorkload)(nil)
-	ed25519HexKeys                             = []string{
-		"323b1d8f4eed5f0da9da93071b034f2dce9d2d22692c172f3cb252a64ddfafd01b057de320297c29ad0c1f589ea216869cf1938d88c9fbd70d6748323dbf2fa7", //nolint:lll
-		"8a7be2e0c9a2d09ac2861c34326d6fe5a461d920ba9c2b345ae28e603d517df148735063f8d5d8ba79ea4668358943e5c80bc09e9b2b9a15b5b15db6c1862e88", //nolint:lll
-	}
-	ed25519PrivKeys      = make([]ed25519.PrivateKey, len(ed25519HexKeys))
-	ed25519Addrs         = make([]codec.Address, len(ed25519HexKeys))
-	ed25519AuthFactories = make([]*auth.ED25519Factory, len(ed25519HexKeys))
-)
-
-func init() {
-	for i, keyHex := range ed25519HexKeys {
-		privBytes, err := codec.LoadHex(keyHex, ed25519.PrivateKeyLen)
-		if err != nil {
-			panic(err)
-		}
-		priv := ed25519.PrivateKey(privBytes)
-		ed25519PrivKeys[i] = priv
-		ed25519AuthFactories[i] = auth.NewED25519Factory(priv)
-		addr := auth.NewED25519Address(priv.PublicKey())
-		ed25519Addrs[i] = addr
-	}
-}
-
```

Comment:
removing this and the genesis configuration is an improvement imo. VM developers are probably going to be fine with these defaults, and if not we can expose a way for them to customize. 

Still some more work to be done cleaning up the `workloadfactory` or removing it all together.